examples/hybrid/sphere/deformation_flow.jl

```diff
@@ -197,6 +215,10 @@ function rhs!(dydt, y, parameters, t, alpha, beta)
         bottom = Operators.ThirdOrderOneSided(),
         top = Operators.ThirdOrderOneSided(),
     )
+    FCTZalesak = Operators.FCTZalesak(
```

Comment:
This test case was for BB. Could we do things incrementally please so that later we know how the Zalesak one compares with BB?

Reply:
We can Zalesak next

Reply:
Sure, you can change Zalesak to BB, it should work, you need check dz in BB too